dNet/dServer.cpp

```diff
@@ -39,7 +39,7 @@ class ReplicaReceiever : public ReceiveDownloadCompleteInterface {
 	}
 } ReceiveDownloadCompleteCB;
 
-dServer::dServer(const std::string& ip, int port, int instanceID, int maxConnections, bool isInternal, bool useEncryption, Logger* logger, const std::string masterIP, int masterPort, ServerType serverType, dConfig* config, bool* shouldShutdown, unsigned int zoneID) {
+dServer::dServer(const std::string& ip, int port, int instanceID, int maxConnections, bool isInternal, bool useEncryption, Logger* logger, const std::string masterIP, int masterPort, ServerType serverType, dConfig* config, Game::signal_t* lastSignal, unsigned int zoneID) {
```

Comment:
Using Game namespace here kind of defeats the purpose of us passing a bunch if variables down to server, but that is something to fix later.